### PR TITLE
update support Kubernetes version

### PR DESCRIPTION
**Pull request by kvalliyurnatt**
Update supported Kubernetes versions for the upcoming release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ Current features:
 
 Supported versions:
 
-*  Kubernetes 1.25-1.29
-*  OpenShift 4.11-4.14
+*  Kubernetes 1.26-1.30
```
Comment (naemono) [inline README.md:20]: Without being able to test on 1.30 (no kind release available quite yet), I'm not sure we can say we support it...

**Comment by naemono** [comment]
buildkite test this -f p=ocp,TESTS_MATCH=TestSmoke -m s=8.13.2

File: README.md (hunk 1 above)
Comment (kvalliyurnatt) [reply README.md:20]: @naemono good point, do we just move it back until a Kind version is available ? I can redo this when that becomes available

Comment (naemono) [reply README.md:20]: Yeah I would say just wait on this until we have a good version of it available and update then... Ensuring a full e2e run in this PR

Comment (kvalliyurnatt) [reply README.md:20]: I updated with the image built by @barkbay, please take another look. TIA

**Comment by kvalliyurnatt** [comment]
buildkite test this -f p=kind,TESTS_MATCH=TestSmoke -m s=8.13.2

**Comment by kvalliyurnatt** [comment]
buildkite test this -f p=kind,DEPLOYER_KIND_NODE_IMAGE=barkbay/node:v1.30.0-dirty@sha256:03aae54e5ce12a2ec3220d5329a58130f9328d9e5b995c8eb4eb0807569bffbe,TESTS_MATCH=TestSmoke -m s=8.13.2

**Review by naemono — APPROVED** [review]
Since the OCP test, and the Kind test both worked, I'd say this is good now. 👍

**Pull request merged**